modules/couchbase/src/main/java/org/testcontainers/containers/CouchbaseContainer.java

```diff
@@ -183,8 +183,8 @@ public SELF withNewBucket(BucketSettings bucketSettings) {
         return self();
     }
 
-
-    protected void initCluster() {
+    @Override
+    protected void containerIsStarted(InspectContainerResponse containerInfo) {
```

Comment:
I think this code should go to "containerIsStarting", and only the part after waitStrategy - to `containerIsStarted`. Then you can just define `setWaitStrategy()` as usual